cucumber/features/careerPage.feature

```diff
@@ -2,3 +2,30 @@ Feature: EPAM job searching
   As a Job searcher
   I want to browser through EPAM Job offers by various criteria
```

Comment:
Type in "browser". It should be "browse".

Reply:
done